index.js

```diff
@@ -0,0 +1,201 @@
+const topscreen = document.getElementById("topscreen");
+const topscreen2 = document.getElementById("topscreen2");
+const quizCa = document.getElementById("category");
+const quizDi = document.getElementById("difficulty");
+const quizQu = document.getElementById("question");
+const answer = document.getElementById("answer");
+
+class Quiz {
+  constructor(quizData) {
+    this.quizData = quizData;
+  }
+  getCategory(num) {
+    return this.quizData[num].category;
+  }
+  getType(num) {
+    return this.quizData[num].type;
+  }
+  getDifficulty(num) {
+    return this.quizData[num].difficulty;
+  }
+  getQuestion(num) {
+    return this.quizData[num].question;
+  }
+  getCorrect_answer(num) {
+    return this.quizData[num].correct_answer;
+  }
+  getIncorrect_answers(num) {
+    return this.quizData[num].incorrect_answers;
+  }
+}
+
+const start = () => {
+  topscreen.innerHTML = "ようこそ";
+  // topscreen2.innerHTML = "以下のボタンをクリック";
+  quizQu.innerHTML = "以下のボタンをクリック";
+  const s_button = document.createElement("div");
+  answer.appendChild(s_button);
+  const s_button2 = document.createElement("button");
+  s_button.appendChild(s_button2);
+  s_button2.innerHTML = "スタート";
+  s_button2.style.backgroundColor = "lightgray";
+  s_button2.style.margin = "20px 0px";
+  s_button2.style.padding = "4px 4px";
+
+  return s_button2;
+};
+
+start().addEventListener("click", () => {
+  let num = 0;
+  let co_n = 0;
```

Comment:
```suggestion
  let correctAnswerNum= 0;
```
toalCorrectAnswesCountなども候補に入ります！